externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/session/HiveSessionImpl.scala

```diff
@@ -54,6 +56,19 @@ class HiveSessionImpl(
     super.runOperation(operation)
   }
 
+  override def getInfo(infoType: TGetInfoType): TGetInfoValue = withAcquireRelease() {
+    infoType match {
+      case TGetInfoType.CLI_SERVER_NAME => TGetInfoValue.stringValue("Hive")
+      case TGetInfoType.CLI_DBMS_NAME => TGetInfoValue.stringValue("Apache Hive")
+      case TGetInfoType.CLI_DBMS_VER => TGetInfoValue.stringValue(HiveVersionInfo.getVersion)
```

Comment:
The intention here is to return the engine information when `kyuubi.server.info.provider=ENGINE`

Reply:
For Hive engine, the engine is something like HiveServer2

Reply:
the code you commented on is from hive engine, it returns Hive version. For Spark, it always returns Spark runtime version.

Reply:
I think the version should be always consistent between Hive client/server, because the server is launched by Kyuubi, using the same jars. 

For Flink, you are right, until Application mode is supported, it always returns the client version.

Reply:
@Yikf FYI, I updated the Flink version description in the last comment.

Reply:
Your concern would happen on JDBC engine and Trino engine since Kyuubi only manages clients of them.

Reply:
You bring up a good question, there are key differences in Kyuubi engines implementation. I have an unpublished article describes them.
https://docs.google.com/document/d/1Qg5zMYTBDzoheUMrZZQF9rqeT8_Occ8HV2bp4mRTKOs/edit#heading=h.7eq9047g3z1t